src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.common;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+	
+	public final Object NULL = null;
+	public final Object NOT_NULL = "TEST";
+	
+	
+	//Test for isAnyNull
+	@Test
+	public void emptyInput(){
+		assertEquals(false, Utils.isAnyNull());
+	}
+	@Test
+	public void oneNullInput(){
```

Comment:
Note that we have a special naming convention for test method names. Please refer coding standard for more details